src/MailClient.php

```diff
@@ -52,12 +51,6 @@ public function sendEmail()
             $email = new PatronEmail($streamData);
         }
 
-        if ($streamData instanceof NewPatron && in_array($streamData->patronType, EDUCATOR_PATRON_TYPES)) {
-            APILogger::addDebug('Sending email using NewPatron object');
-
-            $email = new MyLibraryNycEmail($streamData);
-        }
```

Comment:
It looks like `$email` is initialized using the default new-patron template first and then overridden with the mylibrarynyc template when the ptype indicates they're an educator. The proposed change simply removes the override. Does that mean that educators will get the default new-patron email?

Reply:
I see now that the app sends email to events of type `Patron` and to events of type `NewPatron` (but only when ptype is 149 or 151. By removing the `NewPatron` check, this app will only send emails to events of type `Patron`. I believe the only thing generating a `Patron` event was [the v0.1 endpoint of dgx-patron-creator](https://github.com/NYPL/dgx-patron-creator-service/blob/bbce53595d957ee2286e307fef3d7dc6eee1df71/config/deploy_production.env#L35). If no app is hitting that endpoint, then we can safely retire this whole app, right?